tests/test_request_overrider.js

```diff
@@ -625,23 +634,20 @@ test('Request with `Expect: 100-continue` triggers continue event', t => {
     headers: { Expect: '100-continue' },
   })
 
-  let gotResponse = false
+  const onResponse = sinon.spy()
 
   req.on('continue', () => {
-    t.pass()
-
     // This is a confidence check. It's not really possible to get the response
     // until the request has matched, and it won't match until the request body
     // is sent.
-    t.false(gotResponse)
+    expect(onResponse).not.to.have.been.called()
 
     req.end(exampleRequestBody)
   })
 
   req.on('response', res => {
-    t.is(res.statusCode, 200)
-
-    gotResponse = true
+    onResponse()
+    expect(res.statusCode).to.equal(200)
```

Comment:
This test isn't passing right now because the `res.on('end')` callback is never reached. I was going to move the `scope.done()` and `t.end()` into this block, though it's probably better to figure out why the response isn't being ended and fix it in a more robust way. I'm not sure why it's not being called.

Reply:
It was because there was no `'data'` event handler. This is working now.